build.gradle

```diff
@@ -241,6 +242,18 @@ configure(publishProjects) { Project subproject ->
                     }
                 }
             }
+
+
+        }
+        repositories {
+            maven {
+                url = project.findProperty('maven.repo.s3')
+                credentials(AwsCredentials) {
+                    accessKey "${System.getenv('AWS_ACCESS_KEY_ID')}"
+                    secretKey "${System.getenv('AWS_SECRET_ACCESS_KEY')}"
+                    sessionToken "${System.getenv('AWS_SESSION_TOKEN')}"
+                }
+            }
         }
```

Comment:
But what if `maven.repo.s3` is undefined? Doesn't this break publishing just to Artifactory? I was expecting:
```gradle
if (hasProperty('maven.repo.s3')) {
    repositories {
        maven {
            url = getProperty('maven.repo.s3')
            ...
        }
    }
}
```

I believe you can also assign a name to this repository:
```gradle
maven {
    name = 'AWS'
    url = ...
}
```